examples/benchmark.py

```diff
@@ -389,9 +389,10 @@ def runOneTest(testName, options):
         mm.LocalEnergyMinimizer.minimize(context, 100*unit.kilojoules_per_mole/unit.nanometer)
     context.setVelocitiesToTemperature(temperature)
 
-    # Take one step (to enforce constraints and trigger kernel compulation) and store the initial state
-    integ.step(1)
-    state = context.getState(getPositions=True, getVelocities=True, getEnergy=True, getForces=True, getParameters=True)
+    if options.serialize:
+        # Take a step to enforce constraints on positions and velocities if needing to serialize for Folding@home
+        integ.step(1)
+        state = context.getState(getPositions=True, getVelocities=True, getEnergy=True, getForces=True, getParameters=True)
```

Comment:
If constraints are what you're concerned about, call `context.applyConstraints()`.  Taking a step and retrieving a state aren't necessary.

We just called `setVelocitiesToTemperature()` in the previous line, which already applies constraints to velocities.

Reply:
After applying position constraints, the velocity constraints will no longer be accurate, will they?